config/locales/en/tags/edit.yml

```diff
@@ -5,3 +5,19 @@ en:
       description: Add tags which describe what the content is about. Content will appear in lists on GOV.UK based on each tag.
       api_down: This content can’t be edited right now. We’re having trouble getting the data we need for you to make changes on this page.
       organisation_warning: Access to this document is restricted. Adding or removing organisation tags will change who can access it.
+    types:
```

Comment:
This was a bit of a surprise, and not really consistent with the pattern for these files, since these strings are also used in the document summary. I think we should keep these locales close to the document type definitions for now, so they can participate in the same 'defaulting' mechanism we're aiming for in general. We may want to do more work with the locales, but this change is in excess of what we need to do for the card, and I'm not sure if it will cause problems for the upcoming work.

Reply:
I wasn't too sure where to put these, so would appreciate your thoughts on it. When I last showed you the code, it lived in `config/locales/en/document_types.yml`, which we agreed wasn't a great idea as locale files typically only have one top-level key, not two.

I don't think it should continue to live in `config/document_types.yml` as that means we'd have hardcoded English outside of the I18n namespace. My understanding is that the primary aim of the card is to move all associated English into I18n.

So it looks like we're moving towards having these in a new file: `config/locales/en/tags.yml`. Does that sound right?

Reply:
Note to self after call with Ben: tags & fields can be treated as the same thing, as per the gist.